src/specfem3D/read_adjoint_sources_ASDF.f90

```diff
@@ -130,7 +120,7 @@ subroutine check_adjoint_sources_ASDF(irec, nadj_sources_found)
 
     ! updates counter for found files
     nadj_sources_found = nadj_sources_found + 1
-
+   
```

Comment:
Try to avoid adding whitespace as then BuildBot needs to make an extra commit fixing it.